lib/linux_admin/network_interface.rb

```diff
@@ -112,7 +114,11 @@ def netmask6(scope = :global)
     #
     # @return [String] IPv4 gateway address
     def gateway
-      @network_conf[:gateway]
+      @network_conf[:gateway4]
+    end
+
```

Comment:
Good find! I think a follow up would be good for this. It would be a general enhancement around making things work for multi-nic setups.